addons/web/static/src/scss/base_document_layout.scss

```diff
@@ -1,150 +1,38 @@
-.o_document_layout .o_group {
-    display: flex;
-    .o_document_layout_company {
-        flex: 5;
-        @include media-breakpoint-up(sm, $o-extra-grid-breakpoints) {
-            max-width: 50%;
-        }
-        img {
-            max-height: 100px;
-        }
-        .o_document_layout_colors {
-            vertical-align: middle;
-            display: flex;
-            flex-direction: row;
-            align-items: center;
-            .o_field_widget {
-                width: 30px;
-                margin: 0 5px 0 0;
-            }
-            .btn {
-                padding: 0;
-                margin-left: 10px;
-                .o_form_label {
-                    height: 30px;
-                    font-size: 18px;
-                    margin: 0;
-                    cursor: pointer;
-                }
-            }
-        }
-        select.o_input {
-            height: 25px;
-        }
+/**
+    Couple a variables to make size changes dynamic.
+ */
+$preview_height: 1123;
+$preview_width: 794;
+$preview_scale: 0.50;
 
-    }
-    .o_document_layout_preview {
-        flex: 4;
-        border: 1px solid;
-        border-color: gray;
-        margin-right: 12px;
-        padding: 12px;
-        display: flex;
-        min-height: 420px;
-        flex-direction: column;
-    }
+#o_preview_iframe_wrapper {
```

Comment:
The style definition of the IframeWrapper widget should not be here. You can move it to the `special_fields.scss` file that I mentionned earlier